static/build/miyoo/app/lang/pt-BR.lang

```diff
@@ -50,39 +50,39 @@
     "47": "segundos",
     "48": "em",
     "49": "Lista recente",
-    "50": "Limpar",
+    "50": "Limpar lista",
     "51": "Abrir",
-    "52": "Apagar",
+    "52": "Remover da lista",
     "53": "Atualizar",
     "54": "Atualizar ROMs",
     "55": "Adicionar aos favoritos",
-    "56": "Atualização",
+    "56": "Atualizar",
     "57": "Atualizar o sistema?",
-    "58": "O cartão SD está se esgotando\n%dMB restantes",
+    "58": "Armazenamento baixo\n%dMB livres",
     "59": "Modelo",
     "60": "N.o de série",
     "61": "Versão do firmware",
     "62": "CPU",
     "63": "%dMHz",
     "64": "RAM",
     "65": "%dMB",
-    "66": "Cartão SD",
+    "66": "Armazenamento utilizado/total",
```

Comment:
Personally I think that this should still say "Cartão SD" for consistency since that is how we talk about storage everywhere else in this translation.

Reply:
In this case, I think it should be clear about the available memory instead of using the generic term referring to the physical SD device. Unfortunately, I don't have the device here to check for context.